src/ray/object_manager/plasma/store.cc

```diff
@@ -136,6 +136,12 @@ PlasmaStore::PlasmaStore(instrumented_io_context &main_service, std::string dire
           []() { return absl::GetCurrentTimeNanos(); }) {
   store_info_.directory = directory;
   store_info_.hugepages_enabled = hugepages_enabled;
+
+  RAY_CHECK(
+      RayConfig::instance().maximum_object_store_reservation_for_pulled_objects() > 0 &&
```

Comment:
Why don't we set this to a private attribute? It seems like we access this at 3 places.